doc/sphinx/Pacemaker_Explained/advanced-options.rst

```diff
@@ -1,776 +1,766 @@
+   |                | ``node-health-base`` (this allows resources to start     |
+   |                | even if some attributes are ``yellow``). This strategy   |
+   |                | gives the administrator finer control over how important |
+   |                | each value is.                                           |
+   +----------------+----------------------------------------------------------+
+   | custom         | .. index::                                               |
+   |                |    single: node-health-strategy; custom                  |
+   |                |    single: custom; node-health-strategy value            |
+   |                |                                                          |
+   |                | Track node health attributes using the same values as    |
+   |                | ``progressive`` for ``red``, ``yellow``, and ``green``,  |
+   |                | but do not take them into account. The administrator is  |
+   |                | expected to implement a policy by defining :ref:`rules`  |
+   |                | referencing node health attributes.                      |
+   +----------------+----------------------------------------------------------+
+
+
+Measuring Node Health
+_____________________
+
+Since Pacemaker calculates node health based on node attributes, any method
+that sets node attributes may be used to measure node health. The most common
+are resource agents and custom daemons.
+
+Pacemaker provides examples that can be used directly or as a basis for custom
+code. The ``ocf:pacemaker:HealthCPU``, ``ocf:pacemaker:HealthIOWait``, and
+``ocf:pacemaker:HealthSMART`` resource agents set node health attributes based
+on CPU and disk status.
+
+In order to take advantage of this feature, add the resource to your cluster,
+preferably as a cloned resource to constantly measure health on all nodes.
+For example:
+
+.. topic:: Example HealthIOWait resource configuration
+
+   .. code-block:: xml
+
+      <clone id="resHealthIOWait-clone">
+        <primitive class="ocf" id="HealthIOWait" provider="pacemaker" type="HealthIOWait">
+          <instance_attributes id="resHealthIOWait-instance_attributes">
+            <nvpair id="resHealthIOWait-instance_attributes-red_limit" name="red_limit" value="30"/>
+            <nvpair id="resHealthIOWait-instance_attributes-yellow_limit" name="yellow_limit" value="10"/>
+          </instance_attributes>
+          <operations>
+            <op id="resHealthIOWait-monitor-interval-5" interval="5" name="monitor" timeout="5"/>
+            <op id="resHealthIOWait-start-interval-0s" interval="0s" name="start" timeout="10s"/>
+            <op id="resHealthIOWait-stop-interval-0s" interval="0s" name="stop" timeout="10s"/>
+          </operations>
+        </primitive>
+      </clone>
+
+This way ``attrd_updater`` will set proper status for each node running this
+resource. Any attribute matching "#health-[a-zA-z]+" will force the cluster to
```

Comment:
Putting the regex in monospace might be nice to set it off from the surrounding text.

Reply:
Good point, and that made me realize it didn't quite look right ... sure enough, all we check is that it starts with "#health" (we don't even check for the dash currently).

Reply:
Updated and reworded